src/routers/project.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+
+const router = express.Router();
+
+const project = require('../controllers/project')
+
+router.get('/project', project.getAllProjects)
+router.get('/project/:id', project.getProjectByID)
+router.get('/project/:projectid/comment/', project.getAllComments)
```

Comment:
- here its :id for project since in the controller its written as id for project or if you want to make it clearer change all id project in the controller to projectID
- and it can be /comments not /comment since we are accessing the all comments endpoint
![image](https://user-images.githubusercontent.com/14183741/151787335-7484728f-a7af-42b2-ab33-ebd64e3677e2.png)

Reply:
Changed in all of them.